tests/lib/rules/indent.js

```diff
@@ -7633,6 +7760,80 @@ ruleTester.run("indent", rule, {
             `,
             errors: expectedErrors([[2, 4, 2, "Identifier"], [3, 4, 6, "Identifier"]])
         },
+        {
+            code: unIndent`
+                const foo = a.b(),
+                    longName
+                    = (memberFunction(
+                            'bar',
+                            'bar'
+                        ));
+            `,
+            output: unIndent`
+                const foo = a.b(),
+                    longName
+                    = (memberFunction(
+                        'bar',
+                        'bar'
+                    ));
+            `,
+            errors: expectedErrors([[4, 8, 12, "String"], [5, 8, 12, "String"], [6, 4, 8, "Punctuator"]])
+        },
+        {
+            code: unIndent`
+                const foo = a.b(),
+                    longName =
+                    (memberFunction(
+                            'bar',
+                            'bar'
+                        ));
+            `,
+            output: unIndent`
+                const foo = a.b(),
+                    longName =
+                    (memberFunction(
+                        'bar',
+                        'bar'
+                    ));
+            `,
+            errors: expectedErrors([[4, 8, 12, "String"], [5, 8, 12, "String"], [6, 4, 8, "Punctuator"]])
+        },
+        {
+            code: unIndent`
+                const foo = a.b(),
+                    longName
+                        =memberFunction(
```

Comment:
I think this function name might be confusing since the function callee is just an identifier, not part of a MemberExpression (either with this as the object, or some other object). Could we change it just to like `func` or `doSomething` or something else?